userguide/.htmltest.yml

```diff
@@ -0,0 +1,8 @@
+DirectoryPath: public
+CheckDoctype: false # Sadly, this is false only because of `static/google*.html`
+IgnoreAltMissing: true # FIXME
+IgnoreDirectoryMissingTrailingSlash: true # FIXME
+IgnoreDirs: [_print] # FIXME
+IgnoreEmptyHref: true # FIXME
+IgnoreInternalEmptyHash: true # FIXME
```

Comment:
The options with FIXME tags should eventually be addressed by performing the necessary cleanup in the repo.